lib/parallel_cucumber/cli.rb

```diff
@@ -19,7 +19,8 @@ class Cli
       n: 0, # Default: computed from longest list in json parameters, minimum 1.
       queue_connection_params: ['redis://127.0.0.1:6379', DateTime.now.strftime('queue-%Y%m%d%H%M%S')],
       worker_delay: 0,
-      test_command: 'cucumber'
+      test_command: 'cucumber',
+      backup_worker_count: 1
     }.freeze
```

Comment:
Make it 0 to keep default behaviour the same as before

Reply:
changed